source/administration/config-settings.rst

```diff
@@ -3108,6 +3131,30 @@ Prefix on the Elasticsearch index name. Enables the use of Mattermost Elasticsea
 | This feature's ``config.json`` setting is ``"IndexPrefix": ""`` with string input      |
 +----------------------------------------------------------------------------------------+
 
+Live Indexing Batch Size
```

Comment:
Do any of the new settings need to be mentioned in the elasticsearch docs? Or maybe we add a link to the config settings page somewhere in https://docs.mattermost.com/deployment/elasticsearch.html

Reply:
Added a note, thanks